lib/account_profile/element/first_img_section.dart

```diff
@@ -2,23 +2,57 @@ import 'dart:ui';
 
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter/painting.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:lab_2_try/actions.dart' as action;
+import 'package:lab_2_try/main_view/home.dart';
+
+import '../../states.dart';
 
 Widget _userIcon(double orientationWidth) {
   return Container(
     width: orientationWidth / 4,
     height: orientationWidth / 4,
-    margin:
-        const EdgeInsets.only(left: 50.0, right: 28.0, top: 2.0, bottom: 10.0),
     decoration: BoxDecoration(
         border: Border.all(
           color: Colors.indigo,
         ),
         borderRadius: const BorderRadius.all(Radius.circular(100)),
         image: const DecorationImage(
-            image: AssetImage("assets/images/img_fallout.png"), fit: BoxFit.cover)),
+            image: AssetImage("assets/images/img_fallout.png"),
+            fit: BoxFit.cover)),
   );
 }
 
+Widget _userIconWithStatus(double orientationWidth) {
+  return StoreConnector<StatusState, String>(
+      converter: (store) => store.state.status.toString(),
+      builder: (context, viewModel) {
+        return Container(
+          width: orientationWidth / 3,
+          height: orientationWidth / 4,
+          margin: const EdgeInsets.only(
+              left: 50.0, right: 28.0, top: 2.0, bottom: 10.0),
+          child: Stack(children: [
+            _userIcon(orientationWidth),
+            Align(
+              alignment: Alignment.bottomRight,
+              child: Container(
+                margin: const EdgeInsets.only(bottom: 5, right: 35),
+                width: 25,
+                height: 25,
+                decoration: BoxDecoration(
+                    color: (viewModel == "true"
```

Comment:
don't use `viewModel`, it's too generic, use `isOnline` or similar. also, don't use `"true"`, use real boolean `true` instead